conventional-changelog-emoji/src/conventional-recommended-bump.js

```diff
@@ -1,34 +1,38 @@
 "use strict";
-const fs = require('fs');
-const path = require('path');
+const fs = require("fs");
```

Comment:
Small thing, can we change our prettier settings to use single quotes to reduce the noise here? I also find them ever so slightly faster to type.

Reply:
I understand this, but if you can type single quotes and prettier will automatically switch it to double quotes on commit 😉 

(I'm fine changing to single quotes, but it will actually increase noise, as I've already altered the rest of the repo— I think this file was missed because I didn't run it on js files 🙃 )

Anyhoo, I think this folder should be removed and replaced with a dedicated repository instead of being hidden in here!

Reply:
Linting rules in other projects we work on use single quotes instead of double. It seems much easier to update the prettier definition here than update all of the other repos (one advantage of using prettier!).